### PR TITLE
Added Template for Sections

### PR DESCRIPTION
**Pull request by devendra-dantal04**
Task Performed  :
1) Added Controller for Section Route
2) Changes the path of `config/.env` to `configs/.env` which is typo error in app.js
3) Integrated the controllers with the routes in `routes/sections.js`  file
4) Added json middleware to accept the json data
#1

### CODE REVIEW

File: modules/sections.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        let sections;
+        
+        if(!sections) {
+            res.status(404).send({msg:'Sorry, there are no sections !',  data:null})
+        }
+
+        res.status(200).send({msg:'Successful',  data:sections})
+        
+    }catch (err) {
+        res.status(500).send({msg:'Internal Server Error',  err : err.msg})
```
Comment (hello-work) [inline modules/sections.js:17]:
```suggestion
        res.status(500).send({msg:'Internal Server Error',  err : err.message})     
```

File: modules/sections.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        let section;
+
+        if(!section) {
+            res.status(400).send({msg:'Sections not created', data: null})    
+        }
+
+        res.status(200).send({msg:'Successful',  data:section})
+        
+    }catch (err) {
+        res.status(500).send({msg:'Internal Server Error',  err : err.msg})
```
Comment (hello-work) [inline modules/sections.js:40]:
```suggestion
        res.status(500).send({msg:'Internal Server Error',  err : err.message})     
```

File: modules/sections.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        let section;
+
+        if(!section) {
+            res.status(404).send({msg:'Section Not Found',  data:null})
+        }
+
+        res.status(200).send({msg:'Successful',  data:section})
+        
+    }catch (err) {
+        res.status(500).send({msg:'Internal Server Error',  err : err.msg})
```
Comment (hello-work) [inline modules/sections.js:61]:
```suggestion
        res.status(500).send({msg:'Internal Server Error',  err : err.message})     
```

File: modules/sections.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    const {id} = req.params
+
+    try {
+        // TODO : database call
+        let section;
+
+        res.status(200).send({msg:'Successful',  data:section})
+        
+    }catch (err) {
+        res.status(400).send({msg:'Unsuccessful',  err : err.msg})
```
Comment (hello-work) [inline modules/sections.js:79]:
```suggestion
        res.status(400).send({msg:'Unsuccessful',  err : err.message})     
```

File: modules/sections.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    const {id} = req.params
+
+    try {
+        // TODO : database call
+        let section;
+
+        res.status(200).send({msg:'Successful'})
+        
+    }catch (err) {
+        res.status(400).send({msg:'Unsuccessful',  err : err.msg})
```
Comment (hello-work) [inline modules/sections.js:95]:
```suggestion
        res.status(400).send({msg:'Unsuccessful',  err : err.message})     
```

**Review by hello-work — APPROVED** [review]

**Comment by hello-work** [comment]
@devendra-dantal04 suggestion for you.

- when you raising a PR try to add only one commit.
- After you done your changes for second time you can follow these steps for make it to only one commit

1. ` git reset --soft HEAD~1`
2.  `git add .`
3. `git commit -m <message>`
4.  `git push -f origin <branch_name>`

**Comment by devendra-dantal04** [comment]
> @devendra-dantal04 suggestion for you.
> 
> * when you are raising a PR try to add only one commit.
> * After you have done your changes for the second time you can follow these steps to make it to only one commit
> 
> 1. ` git reset --soft HEAD~1`
> 2. `git add .`
> 3. `git commit -m <message>`
> 4. `git push -f origin <branch_name>`

Thank you @hello-work for this advice 😊  I will follow these.

**Pull request merged**